libs/webb-ui-components/src/components/Radio/RadioItem.tsx

```diff
@@ -23,33 +23,37 @@ export const RadioItem = forwardRef<HTMLDivElement, RadioItemProps>(
     return (
       <div
         {...props}
-        className={twMerge('flex items-center space-x-2', className)}
+        className={twMerge('flex items-center space-x-1', className)}
         ref={ref}
       >
-        <RadioGroup.Item
-          {...overrideRadixRadioItemProps}
-          className={twMerge(
-            'peer w-[18px] h-[18px] rounded-full bg-mono-0 dark:bg-mono-180 border border-mono-100',
-            'enabled:radix-state-unchecked:hover:bg-blue-10 enabled:radix-state-unchecked:hover:dark:bg-blue-120',
-            'enabled:radix-state-unchecked:hover:border-blue-40 enabled:radix-state-unchecked:hover:dark:border-blue-90',
-            'enabled:radix-state-unchecked:hover:shadow-[0_0_0_1px_rgba(213,230,255,1)] hover:dark:shadow-none',
-            'radix-state-checked:border-2 radix-state-checked:border-blue-70 dark:radix-state-checked:border-blue-50',
-            'radix-disabled:border-mono-80 dark:radix-disabled:border-mono-120 dark:radix-disabled:bg-mono-140',
-            overrideRadixRadioItemProps?.className,
-          )}
-          value={value}
-          id={id}
-        >
-          <RadioGroup.Indicator
-            {...overrideRadixRadioIndicatorProps}
+        <div className="size-[36px] group rounded-full hover:bg-[#5953F91A] hover:dark:bg-[#5953F933] flex items-center justify-center has-[:disabled]:hover:!bg-transparent has-[:radix-state-checked]:!bg-[#5953F933] has-[:radix-state-checked]:dark:!bg-[#5953F94D]">
+          <RadioGroup.Item
+            {...overrideRadixRadioItemProps}
             className={twMerge(
-              'flex items-center justify-center w-full h-full relative',
-              "after:content-[''] after:block after:w-2.5 after:h-2.5 after:rounded-[50%]",
-              'after:bg-blue-70 after:dark:bg-blue-50',
-              overrideRadixRadioIndicatorProps?.className,
+              'peer group size-[18px] rounded-full border-[2px] border-mono-100 dark:border-mono-100',
+              'radix-disabled:border-mono-60 dark:radix-disabled:border-mono-140',
+              'radix-disabled:radix-state-checked:border-mono-60 dark:radix-disabled:radix-state-checked:border-mono-140',
+              'radix-state-checked:border-blue-60 dark:radix-state-checked:border-blue-40',
+              'radix-state-checked:hover:border-blue-50 dark:radix-state-checked:hover:border-blue-50',
+              overrideRadixRadioItemProps?.className,
             )}
-          />
-        </RadioGroup.Item>
+            value={value}
+            id={id}
+          >
+            <RadioGroup.Indicator
+              {...overrideRadixRadioIndicatorProps}
+              className={twMerge(
+                'flex items-center justify-center w-full h-full relative',
+                "after:content-[''] after:block after:w-2.5 after:h-2.5 after:rounded-[50%]",
+                'radix-state-checked:after:bg-blue-60 radix-state-checked:after:dark:bg-blue-40',
```

Comment:
I noticed some inconsistencies with checkboxes here:
<img width="1065" alt="Screenshot 2024-07-31 at 10 36 49 PM" src="https://github.com/user-attachments/assets/7b082c71-5767-475b-bd6d-ef678021e293">

Reference from [checkbox](https://github.com/webb-tools/webb-dapp/blob/8e56b337d45ebf1d2e83a091e9ecdb6336ea743d/libs/webb-ui-components/src/components/CheckBox/Checkbox.tsx): 
<img width="522" alt="Screenshot 2024-07-31 at 10 44 56 PM" src="https://github.com/user-attachments/assets/6a8a1440-5a7b-41c4-9b13-c6fd45de0421">

`blue-60` & `blue-40` here should be in pressed state, and checked state is `blue-50`